assets/js/modules/analytics/util/index.js

```diff
@@ -156,23 +157,29 @@ function reduceAnalyticsRowsData( rows, selectedStats ) {
  *
  * @since 1.0.0
  *
- * @param {Object} reports       The data returned from the Analytics API call.
- * @param {Array}  selectedStats The currently selected stat we need to return data for.
- * @param {number} days          The number of days to extract data for. Pads empty data days.
+ * @param {Object} reports The data returned from the Analytics API call.
+ * @param {Object} options Extraction options.
  * @return {Array} The dataMap ready for charting.
  */
-export const extractAnalyticsDashboardData = ( reports, selectedStats, days ) => {
+export function extractAnalyticsDashboardData( reports, options ) {
```

Comment:
See above, let's not change this function's signature - the new code shouldn't use it.